### PR TITLE
Windows defender exclusions and missing ova generation fixes

### LINKED ISSUE #508: OVA Windows Image Builder unable to create ovf
**What steps did you take and what happened:**
Trying to build Windows OVA using vmware-local or vsphere builders results the following errors due to changes in the image-build-ova.py script.  I made a change on Error 1 to add the changes to the packer-windows.json file

1. Added in [PR](https://github.com/kubernetes-sigs/image-builder/pull/467)
```
==> vmware-iso (shell-local): Traceback (most recent call last):
==> vmware-iso (shell-local):   File "./hack/image-build-ova.py", line 270, in <module>
==> vmware-iso (shell-local):     main()
==> vmware-iso (shell-local):   File "./hack/image-build-ova.py", line 77, in main
==> vmware-iso (shell-local):     with io.open(args.ovf_template, 'r', encoding='utf-8') as f:
==> vmware-iso (shell-local): FileNotFoundError: [Errno 2] No such file or directory: '../../hack/ovf_template.xml'
Build 'vmware-iso' errored after 38 minutes 1 second: 1 error(s) occurred:

* Post-processor failed: Script exited with non-zero exit status: 1.Allowed exit codes are: [0]
```

2. This seems to have been changed by this [PR](https://github.com/kubernetes-sigs/image-builder/pull/470)
```
==> vmware-iso (shell-local): Traceback (most recent call last):
==> vmware-iso (shell-local):   File "./hack/image-build-ova.py", line 270, in <module>
==> vmware-iso (shell-local):     main()
==> vmware-iso (shell-local):   File "./hack/image-build-ova.py", line 141, in main
==> vmware-iso (shell-local):     'DISTRO_NAME': build_data['distro_name'],
==> vmware-iso (shell-local): KeyError: 'distro_name'
Build 'vmware-iso' errored after 40 minutes 43 seconds: 1 error(s) occurred:

* Post-processor failed: Script exited with non-zero exit status: 1.Allowed exit codes are: [0]
```

**What did you expect to happen:**
Image ova should generate successfully

**Anything else you would like to add:**
I think we need to work out a way to ensure that changes can be reviewed for windows as the builders are different from different Linux distros but are intertwined 

**Environment:**

Project ([Image Builder for Cluster API](https://github.com/kubernetes-sigs/image-builder/tree/master/images/capi)):

Additional info for Image Builder for Cluster API related issues:

- OS (e.g. from `/etc/os-release`, or `cmd /c ver`): Windows
- Packer Version: 
- Packer Provider: OVA
- Ansible Version: 
- Cluster-api version (if using):
- Kubernetes version: (use `kubectl version`):

/kind bug 
/sig windows
[One or more /area label. See https://github.com/kubernetes-sigs/cluster-api/labels?q=area for the list of labels]

### PR DESCRIPTION
**Pull request by perithompson**
What this PR does / why we need it:
This PR adds a required Windows Defender exclusions for containerd.exe and ctr.exe and also fixes some missing updates from recent PRs by registering a distro_name and version and uses the shared OVF template

Which issue(s) this PR fixes #508

**Additional context**

### CODE REVIEW

**Comment by perithompson** [comment]
@jsturtevant and @kkeshavamurthy are you able to review this for me?

**Comment by codenrhoden** [comment]
/lgtm
Definitely will wait on input from others. :)

**Comment by jsturtevant** [comment]
thanks for adding the containerd exclusions!  I am not as familiar with the OVA changes but look ok to me

/lgtm

File: images/capi/packer/ova/packer-windows.json [hunk 1]
```diff
@@ -27,7 +27,10 @@
     "windows_updates_categories": null,
     "wins_url": "https://github.com/rancher/wins/releases/download/v{{user `wins_version`}}/wins.exe",
     "disable_hypervisor" : null,
-    "netbios_host_name_compatibility": null
+    "netbios_host_name_compatibility": null,
+    "distro_name" : null,
+    "distro_version" : null,
+    "distro_arch" : null
```
Comment (kkeshavamurthy) [inline images/capi/packer/ova/packer-windows.json:33]: Setting them to `null` explicitly is probably not needed. I'm all for keeping `packer-*.json` file as short as possible. But I'm ok with keeping if it helps.

Comment (perithompson) [reply images/capi/packer/ova/packer-windows.json:33]: I think if we remove these we probably should remove all of the variables set as null,  one thing I always struggle with is a defined list of all the variables that we can provide per builder/OS and I have to admit this has kind of been my master list for the windows builder, although I'm happy either way, I think there needs to be some clarity on what variables we need to define?  Unless that list is somewhere and I missed it?

Comment (codenrhoden) [reply images/capi/packer/ova/packer-windows.json:33]: I agree we need to be more consistent across the project.

For now, there are no other builders that set `"distro_*" : null`, so lets' be consistent with that and remove those three additions. Then this is ready to merge.

Comment (perithompson) [reply images/capi/packer/ova/packer-windows.json]: 👍 ive also done a sort on the variables in that file for a little tidy up

**Pull request merged**